.circleci/config.yml

```diff
@@ -68,17 +68,17 @@ jobs:
       docker_layer_caching: true
     resource_class: xlarge
     working_directory: ~/experimenter
-    parallelism: 5
+    parallelism: 6
```

Comment:
I assume this is because the `parallel_pytest_args.txt` file now has 6 lines instead of 5?
If so, could/should this be semi-scripted like some `$(wc -l "app/tests/integration/nimbus/parallel_pytest_args.txt")`? Or should we add some comment about the pseudo-magic number?